src/Schema.php

```diff
@@ -46,6 +49,41 @@ public static function uninstall() {
     Admin::removeAccessCapability();
   }
 
+  /**
+   * Ensures dynamic response headers for all requests from htaccess.
+   */
+  public static function ensureResponseHeadersHtaccess() {
+    $template = file_get_contents(Plugin::getBasePath() . '/conf/.htaccess.headers');
+    if (defined('CORE_STANDARDS_CONTENT_SECURITY_POLICY')) {
+      static::findAndReplaceHeader($template, 'Content-Security-Policy', CORE_STANDARDS_CONTENT_SECURITY_POLICY);
+    }
+    if (defined('CORE_STANDARDS_REFERRER_POLICY')) {
+      static::findAndReplaceHeader($template, 'Referrer-Policy', CORE_STANDARDS_REFERRER_POLICY);
+    }
+    if (defined('CORE_STANDARDS_STRICT_TRANSPORT_SECURITY')) {
+      static::findAndReplaceHeader($template, 'Strict-Transport-Security', CORE_STANDARDS_STRICT_TRANSPORT_SECURITY);
+    }
+    if (defined('CORE_STANDARDS_X_CONTENT_TYPE_OPTIONS')) {
+      static::findAndReplaceHeader($template, 'X-Content-Type-Options', CORE_STANDARDS_X_CONTENT_TYPE_OPTIONS);
+    }
+    if (defined('CORE_STANDARDS_X_XSS_PROTECTION')) {
+      static::findAndReplaceHeader($template, 'X-XSS-Protection', CORE_STANDARDS_X_XSS_PROTECTION);
+    }
+    static::createOrPrependFile(ABSPATH . '.htaccess', $template, "\n");
+  }
+
+  /**
+   * Performs a preg match to replace the new header value in the stream.
+   */
+  private static function findAndReplaceHeader(string &$content, string $headerName, string $headerValue) {
```

Comment:
Minor: Local variable names within functions should be in `$snake_case` (as seen in the rest of the file).  Only class properties and methods use lowerCamelCase.
```suggestion
  private static function findAndReplaceHeader(string &$content, string $header_name, string $header_value) {
```